### ISSUE TITLE
Trouble with documents/images after upgrade

### ISSUE THREAD

**Issue by ftrojahn** [danmunn/redmine_dmsf#703]
Hello,

this week we did upgrade from Redmine 3.1.x to 3.3-stable including some plugins,
so dmsf went from 1.5.5 to 1.5.9.

We didn't run the convert rake task(s), since we don't have/want enable DMS on all projects, only few.
Then, we run into troubles uploading documents may be related to #48:

1st: Uploading documents/files did not work, inline pictures are not shown. Workaround was to enable DMS for the project - but we don't really need this everywhere. 

2nd: Then, every change on a file sends e-mail to the project manager, even while creating tickets and attaching files to them. Ticket history is crowded with this, too. Could this be shut off?

So my questions are:

* are we really forced to convert all projects to dmsf?
* if yes, how can we convert the dmsf files back to standard document module to remove dmsf plugin later on
* if not - why do the uploaded files not show up, if dmsf is off - is this a bug, or may be a (rights) setting/config value we didn't notice.

As we need to fix this urgently - is there a way to go one version back e.g. to 1.5.8 or to a certain commit to a point before this behaviour was introduced, so standard document module and dmsf can coexist as they did??

The workaround we found for the user atm is to use only the "file upload" button on ticket creation/comment, but not the "document" upload, as both are shown. But as said, download/inline view is only working if dmsf is active for the project. 

Thank you very much for any advice in advance what we can do here.

**Comment by picman** [comment]
1) There is no need to call the convert rake task. It's supposed to be called just once in case of migration from Redmine Documents module to DMSF module.
2) There is no way back, I'm afraid.
3) There was a mistake that the new feature allowing to attach documents to issues didn't take into account whether DMSF module is active or not. I've fixed it. Just call _git pull_ for the _master_ branch. Moreover, I've added a new plugin option **Act as attachable** which is switched off as default.
